### PR TITLE
MBS-8586: Simplify horizontal scrolling

### PR DESCRIPTION
**Pull request by sh-csg**

### CODE REVIEW

**Review by PhMemmel — CHANGES_REQUESTED** [review]
Looks good in general.

Some minor questions/thoughts/suggestions:
- Would it be a good idea to make the steps when clicking the buttons dependent on the card with? Currently it feels a bit like I have to click too often to move right, so maybe we can use cardwith/2 as scroll length or even 0.9*cardwith or something?
- To fit into the overall design maybe we could style the buttons with rounded corners.
- It would be nice if the buttons could be transparent, so when they are shown, you can still see the content of the card behind them
- User might eventually expect that I can keep scrolling by keeping the buttons pressed. Is this something we want?

**Comment by sh-csg** [comment]
Introducing transparency produces a quite messy output - maybe we'll find another approach for implementing
this. 
Default scrolling distance is now the width of a column, inner corners are rounded.
Whether to keep scrolling when button is constantly pressed is an open question for me - I would move this to another ticket,

File: amd/src/board.js [hunk 1]
```diff
@@ -319,4 +335,35 @@ export default class extends KanbanComponent {
...
+     * Only show scroll buttons if it's possible to scroll in this direction.
+     */
+    _updateScrollButtons() {
+        let main = this.getElement(selectors.MAIN);
+        if (main.scrollLeft <= 1) {
+            this.getElement(selectors.SCROLLLEFT).style.setProperty('visibility', 'hidden');
+        } else {
+            this.getElement(selectors.SCROLLLEFT).style.setProperty('visibility', 'visible');
+        }
+        if (main.scrollLeft < main.scrollLeftMax) {
```
Comment (PhMemmel) [inline amd/src/board.js:363]: Not a standard property, does not work on Chrome and many other browsers, see: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeftMax?retiredLocale=de

Needs to be changed.

File: styles.css [hunk 2]
```diff
@@ -391,3 +391,30 @@ a.mod_kanban_attachment_item {
...
+    border: none;
+}
+
+.mod_kanban_scroll_left {
+    left: 0;
+}
+
+.mod_kanban_scroll_right {
+    left: calc(100vw - 30px);
+}
```
[the remark this answers is not in the document]
Comment (PhMemmel) [reply styles.css:420]: still no newline

**Review by PhMemmel — CHANGES_REQUESTED** [review]
Pls see comments.

**Comment by sh-csg** [comment]
Accidentally pushed wrong version - sorry for that!

**Review by PhMemmel — APPROVED** [review]
Great feature, well done!

Internally reviewed, happy to merge.

Some more improvements regarding button styling and general handling will be introduced in a separate PR.

**Pull request merged**